app/controllers/attendances_controller.rb

```diff
@@ -64,8 +74,16 @@ def set_attendance
       @attendance = Attendance.find(params[:id])
     end
 
+    def set_employee
+      @employee = Employee.find_by private_number: params[:private_number]
```

Comment:
If you add an exclamation mark here `!` the find_by will throw an `ActiveRecord::RecordNotFound` error, so you don't need to add the unless present..., what do you think?

Reply:
I didn't know i will test this, thanks